src/main/java/com/revature/globetrotters/controller/UserAccountController.java

```diff
@@ -1,12 +1,123 @@
 package com.revature.globetrotters.controller;
 
+import java.util.Optional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import com.revature.globetrotters.entity.UserAccount;
+import com.revature.globetrotters.service.AccountService;
 
 @Controller
-@RequestMapping("/accounts")
```

Comment:
We use a RequestMapping annotation with a value of "/users" to automatically prepend "/users" to all endpoints within the class. For example, PostMapping("users/login") -> PostMapping("/login").